Lib/inspect.py

```diff
@@ -3106,10 +3236,10 @@ def __str__(self):
         return rendered
 
 
-def signature(obj, *, follow_wrapped=True, globalns=None, localns=None):
+def signature(obj, *, follow_wrapped=True, globals=None, locals=None, eval_str=ONLY_IF_STRINGIZED):
```

Comment:
Renaming the keyword arguments is definitely unsafe

Reply:
Why?  They are new in 3.10.

Reply:
Sorry, didn't realize they are new.

Reply:
What happened was, those arguments were added to `inspect.signature()` for Python 3.10 as part of the "PEP 563 becoming default behavior" work.  `inspect.signature()` was changed to call  `typing.get_type_hints()`.  When that work was backed out, these changes were overlooked and left in.

I changed the names because I thought they should match the arguments to `eval()`.  If for some reason we decided that `inspect.signature()` shouldn't call `inspect.get_annotations()` after all, the arguments should simply be removed.